assembl/static2/fonts/assembl-glyph.svg

```diff
@@ -34,8 +34,12 @@
 
 <glyph glyph-name="minus-circled" unicode="&#xe80d;" d="M420 770q174 0 297-123t123-297-123-297-297-123-297 123-123 297 123 297 297 123z m252-368l-504 0 0-102 504 0 0 102z" horiz-adv-x="840" />
 
+<glyph glyph-name="up-bold" unicode="&#xe80e;" d="M760 336l-192 0 0-350-376 0 0 350-192 0 380 380z" horiz-adv-x="760" />
```

Comment:
oops, we have 2 branches (this one and piwik metrics) that add new icons to the fontello font, so there will probably be a problem after merge: we will probably have to re-export the fontello font after the 2 merges